### DISCUSSION TITLE
Vlang on Peak

### DISCUSSION THREAD

**Discussion by ghost** [vlang/v#16901] [General]
Can i learn vlang for my future development.why autocompletion is not available in vscode.

**Comment by MatejMagat305** [comment]
So my opinion is:
This language has potencial, but question is whether you want, every language improve yours skill..., for example I have learned 9 P. languages before I introduce pascal, python, c, c++..., but until I have learned golang I do not understand deeply, after that I was be able to look to c++ and have more deep knoledge what is inside..., so learning this is not wasted time ...

Other question is whether vlang will have first rank in stackoverflow ..., it is posible, but the chance that will be true during next 10 years is very low because there are more mature languages and people do not like change language verry offen 

But I wish vlang become at least popular as c#, but is there a long journey ...

**Reply by ghost** [reply]
Probably it right 👍
